info.go

```diff
@@ -391,6 +397,29 @@ func (pi *ProgramInfo) Instructions() (asm.Instructions, error) {
 	return insns, nil
 }
 
+// JitedSize returns the size of the program's JIT-compiled machine code in bytes, which is the
+// actual code executed on the host's CPU. This field requires the BPF JIT compiler to be enable.
+//
+// Available from 4.13. Reading this metadata requires CAP_SYS_ADMIN or equivalent.
+//
+// The bool return value indicates whether the program has been JIT-compiled, or whether this
+// field is available.
+func (pi *ProgramInfo) JitedSize() (uint32, bool) {
+	return pi.jitedSize, pi.jitedSize > 0
```

Comment:
If this could be a permissions error, I would follow the same convention as TranslatedSize:

```suggestion
	return 0, fmt.Errorf("insufficient permissions or unsupported kernel: %w", ErrNotSupported)
```

Reply:
I believe if JIT compiler is disabled, `jited_prog_len` could also be zero.
It passed the `vimto` test for kernel `v4.14`, however, on mainline Ubuntu kernels below `v4.15`, I found the JIT compiler to be disabled by default.
I included "JIT compiler disabled" in the error message as well, although, I'm not sure it would go along with 
`ErrNotSupported`.

Reply:
Perfect, thank you!